community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/checkpoint/CheckPointerImpl.java

```diff
@@ -46,6 +48,9 @@
     private final Log msgLog;
     private final CheckPointTracer tracer;
 
+    private volatile long lastCheckPointedTx;
```

Comment:
I think this field does not need `volatile` because it is always accessed under `lock`